gateway-server/src/test/java/org/apache/knox/gateway/services/security/impl/RemoteAliasServiceTestProvider.java

```diff
@@ -88,7 +88,8 @@ public void removeAliasesForCluster(String clusterName, Set<String> aliases) thr
 
     @Override
     public char[] getPasswordFromAliasForCluster(String clusterName, String alias) {
-      return aliases.getOrDefault(clusterName, new HashMap<>()).get(alias).toCharArray();
+      String value = aliases.getOrDefault(clusterName, new HashMap<>()).get(alias);
+      return value == null ? null : value.toCharArray();
```

Comment:
This is only needed now because the password may be null since it's not ALWAYS being generated?

Reply:
It is because of the new test (this is also test code). We simulate that there is no password yet and check if the password was generated.